v1/plugin/plugin.go

```diff
@@ -200,14 +267,10 @@ func applySecurityArgsToMeta(m *meta, args *Arg) error {
 
 // buildGRPCServer configures and builds GRPC server ready to server a plugin
 // instance
-func buildGRPCServer(typeOfPlugin pluginType, name string, version int, opts ...MetaOpt) (server *grpc.Server, m *meta, err error) {
-	args, err := getArgs()
-	if err != nil {
-		return nil, nil, err
-	}
+func buildGRPCServer(typeOfPlugin pluginType, name string, version int, arg *Arg, opts ...MetaOpt) (server *grpc.Server, m *meta, err error) {
 	m = newMeta(typeOfPlugin, name, version, opts...)
 
-	if err := applySecurityArgsToMeta(m, args); err != nil {
+	if err := applySecurityArgsToMeta(m, arg); err != nil {
```

Comment:
It's confusing that there is `arg` instead `args` - I went through all this change and it seems that getArgs() in  [L204](https://github.com/intelsdi-x/snap-plugin-lib-go/pull/84/files#diff-01c2cbb747cc453d78a7b29d3e51c7d7L204) is just called earlier (e.g. in [startPlugin](https://github.com/intelsdi-x/snap-plugin-lib-go/pull/84/files#diff-01c2cbb747cc453d78a7b29d3e51c7d7R378)) and its output is passed to ` buildGRPCServer`.  Basically, I prefer to keep `args` - but I do not insist on that, it might be `arg` as well, but keep function names consistent with that decision